### PR TITLE
[CI] add ci support.

### PR DESCRIPTION
**Pull request by marvin-Yu**

### CODE REVIEW

**Comment by changqi1** [comment]
LGTM

File: ci_build [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+MODEL() {
+    echo "Running model function with arguments: $@"
+    # DATASETS_LIST=( 'Llama-2-7b' 'chatglm-6b' 'chatglm2-6b' 'llama-13b' 'llama-7b' 'opt-1.3b' 'opt-13b' 'opt-30b' )
+    DATASETS_LIST=( 'llama-2-7b-chat' 'llama-2-13b-chat' 'chatglm2-6b' 'llama-13b' 'llama-7b' )
+    DATATYPE_LIST=( 'fp16' 'bf16' 'int8' 'bf16_fp16' 'bf16_int8' )
+    for DATASET in ${DATASETS_LIST[@]}; do
+    for DATATYPE in ${DATATYPE_LIST[@]}; do
+        OMP_NUM_THREADS=48 numactl -N 0 -m 0 build/example --model=/data/${DATASET}-cpu/ --token=/data/${DATASET}-hf/tokenizer.model --dtype=${DATATYPE} --loop=3 --input_len=32 --output_len=32
```
Comment (changqi1) [inline ci_build:29]: "--input_len=32 --output_len=32" replace with "--input_len=16 --output_len=32 --no_stream"

**Review by changqi1 — APPROVED** [review]

File: ci_build (hunk 1 above)
Comment (changqi1) [reply ci_build:29]: "numactl -N 0 -m 0" repalce with "numactl -N 0 -m 2"

Comment (marvin-Yu) [reply ci_build]: Done!

**Review by heagoo — APPROVED** [review]

**Review by pujiang2018 — APPROVED** [review]

**Pull request merged**